Gems/PhysX/Code/Include/PhysX/EditorColliderComponentRequestBus.h

```diff
@@ -64,6 +64,30 @@ namespace PhysX
         /// @return The height of the capsule collider.
         virtual float GetCapsuleHeight() = 0;
 
+        /// Set the radius of the cylinder collider.
+        /// @param radius The radius of the cylinder collider.
+        virtual void SetCylinderRadius(float radius) = 0;
+
+        /// Gets the radius of the cylinder collider.
+        /// @return The radius of the cylinder collider.
+        virtual float GetCylinderRadius() = 0;
+
+        /// Set the height of the cylinder collider.
```

Comment:
uber nit: Set -> Sets (nicer to use this form for API comments rather than inline comments where the work is actually happening, the Get calls are already doing this which is cool)

Reply:
yeah copy & pasted from capsule & sphere comments above where `Set` is used :) 
fixed those too, thanks!